tracer/test/test-applications/integrations/Samples.WeakHashing/testHash.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Security.Cryptography;
+
+namespace Samples.WeakHashing
+{
+    internal class testHash : HashAlgorithm
```

Comment:
It might be worth having a test where you inherit directly from a vulnerable hash, e.g. 

```csharp
internal class CustomMD5: MD5
{
}
```

Reply:
Sure. Done!